sdcm/nemesis.py

```diff
@@ -1775,29 +1775,31 @@ def disrupt_mgmt_backup(self):
         self._mgmt_backup(backup_specific_tables=False)
 
     def _mgmt_backup(self, backup_specific_tables):
-        # TODO: When cloud backup is supported - add this to the below 'if' statement:
-        #  and not self.cluster.params.get('use_cloud_manager')
-        if not self.cluster.params.get('use_mgmt'):
-            raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
-        if not self.cluster.params.get('backup_bucket_location'):
-            raise UnsupportedNemesis('backup bucket location configuration is not defined!')
-
         mgr_cluster = self.cluster.get_cluster_manager()
-        cluster_backend = self.cluster.params.get('cluster_backend')
-        backup_bucket_location = self.cluster.params.get('backup_bucket_location')
-        if self._is_it_on_kubernetes():
-            bucket_name = f"s3:{backup_bucket_location.split()[0]}"
-        elif cluster_backend == 'aws':
-            bucket_name = f"s3:{backup_bucket_location.split()[0]}"
-        elif cluster_backend == 'gce':
-            bucket_name = f"gcs:{backup_bucket_location}"
+        if self.cluster.params.get('use_cloud_manager'):
+            auto_backup_task = mgr_cluster.backup_task_list[0]
+            location = auto_backup_task.arguments.split('-L')[1].split()[0]
```

Comment:
Please write an example of what the string
Also, I think it better to do it like that
```
            location = auto_backup_task.arguments.split('-L', maxsplit=1)[1].split(maxspit=1)[0]
```

Reply:
added an example. using 'maxsplit=1' will actually find a longer unneeded string.